src/Microsoft.Azure.SignalR.Management/IServiceHubContext.cs

```diff
@@ -12,13 +13,19 @@ namespace Microsoft.Azure.SignalR.Management
     /// </summary>
     public interface IServiceHubContext : IHubContext<Hub>
     {
+        /// <summary>
+        /// Performs a negotiation operation asynchronously that routes a client to a Azure SignalR instance.
+        /// </summary>
+        /// <returns>A negotiation response object that contains an endpoint url and an access token for the client to connect to the Azure SignalR instance. </returns>
+        Task<NegotiationResponse> NegotiateAsync(NegotiationOptions negotiationOptions = null, CancellationToken cancellationToken = default);
```

Comment:
Breaking change

Reply:
Do you think to use abstract class instead of interface to get more flexibility is a good idea? We want to add a new method but don't want to maintain two versions of SDK.

Reply:
Yep, it is if we plan to change it changes often.